style.css

```diff
@@ -59,4 +68,204 @@
   text-align: center;
   margin-top: 50px;
   margin-bottom: 60px;
-}
\ No newline at end of file
+}
+
+.works {
+  margin-top: 0;
+  text-align: center;
+}
```

Comment:
> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

- [ ] You have done great so far :+1: :+1: But there are styling issues and some requirements you need to make before moving to the next milestone, According to the project requirement you are mandated to use CSS GRID to lay out the works section, You didn't implement this in your CSS, I'll suggest you use this in your style.